insights/parsers/lsblk.py

```diff
@@ -138,8 +145,10 @@ def __repr__(self):
             )
         else:
             # As long as the regular expression in LsBlock works, we must end
-            # up with NAME and TYPE records here.
-            assert 'TYPE' in self.data
+            # up with NAME and TYPE records here.  However, the LSBlockPairs
+            # parsing isn't able to be that strict.
+            if 'TYPE' not in self.data:
+                raise ParseException("TYPE not found in LsBlockPairs line")
```

Comment:
@PaulWay its not clear why `__repr__` should raise an exception?  If the data is not parsed correctly shouldn't the exception be thrown before the parser data is returned from `parse_content`?

Reply:
Yeah, that's probably better - I think this may be the first parser that I know of that checks the correctness of the data after all the data has been read (rather than raising a ParseException during the read process).